go/kbfs/simplefs/simplefs.go

```diff
@@ -1164,7 +1172,7 @@ func (k *SimpleFS) SimpleFSMove(ctx context.Context, arg keybase1.SimpleFSMoveAr
 			if err != nil {
 				return err
 			}
-			return k.doRemove(ctx, arg.Src)
+			return k.doRemove(ctx, arg.Src, false)
```

Comment:
Do we plan to support recursive `SimpleFSMove` in the future?

Reply:
Yep!  Working on it right now, as a followup to this PR.  That was the motivation for this PR, actually.